### PR TITLE
Add a step for cert rotation, enable in openshift/oc

### PR DESCRIPTION
**Pull request by cgwalters**
To test out recent PRs there.

### CODE REVIEW

File: ci-operator/jobs/openshift/oc/openshift-oc-master-presubmits.yaml [hunk 1]
```diff
@@ -119,6 +119,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-agnostic-ovn-cmd,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (cgwalters) [inline ci-operator/jobs/openshift/oc/openshift-oc-master-presubmits.yaml:123]: Should be `false`, will fix in the next push

**Comment by cgwalters** [comment]
/pj-rehearse pull-ci-openshift-oc-master-e2e-aws-certrotation

**Comment by cgwalters** [comment]
> INFO[2023-06-15T21:39:49Z] Step e2e-aws-certrotation-openshift-oc-certrotation succeeded after 19m24s. 

:tada:

**Comment by deads2k** [comment]
let's get started!

/lgtm
/approve

**Pull request merged**